recipe/meta.yaml

```diff
@@ -25,15 +20,16 @@ requirements:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
   host:
-    - {{ mpi }}
+    - mpi
     - python
-    - swig 3.0.10
+    - swig 3.0.11
     - doxygen
     - numpy
     - mpi4py
+    - trilinos {{ version }}
 
   run:
-    - {{ mpi }}
+    - mpi
```

Comment:
Should this be `mpich`???

Reply:
I would like it to support mpich or openmpi.

Reply:
Do you need to copy the structure of https://github.com/conda-forge/trilinos-feedstock/tree/master/recipe
or is that recipe too old?

Reply:
https://conda-forge.org/docs/maintainer/knowledge_base.html?highlight=mpi#building-mpi-variants

It seems like you do.

Reply:
My thinking is, that if there are not going to be conflicts, the PyTrilinos structure needs to be as close to the Trilinos structure as possible.  Your linked Trilinos feedstock shouldn't be too old for 12.12.1.

Reply:
I thought we were:
1. Going to remvoe the trilinos dependency
2. Rebuild the trilinos packages from scratch. <- this is happening anyway.

Reply:
There are a lot of balls in the air.  Since the Linux build made it to testing without the compilation error I had seen on MacOS, I left the trilinos dependency in there.  I'm willing to remove it, though.

Reply:
It seems to me like you would be clobbering things. that isn't good.